### PR TITLE
feat(docker+win): fixing launch, adding windows support

### PR DESCRIPTION
**Pull request by BaptisteHubert**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
New dockerfile for deployment on the team server, also added a res folder for documentation

### CODE REVIEW

**Comment by rigelk** [comment]
I don't think the documentation should sit within the repository, but rather in the wiki. That way we also avoid adding binaries.

File: package.json [hunk 1]
```diff
@@ -63,7 +64,7 @@
   },
   "size-limit": [
     {
-      "path": "dist/*es2015*.js"
+      "path": "dis-t/*es2015*.js"
```
Comment (rigelk) [inline package.json]: why the change?

File: scripts/get-assets-w.bat [hunk 2]
```diff
@@ -0,0 +1,19 @@
+@ECHO off
+REM - Script visant à récupérer les assets javascripts utilisés pour le build de l'application.
```
Comment (rigelk) [inline scripts/get-assets-w.bat]: comments should be in english

File: scripts/get-assets-w.bat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+CALL :RecuperationAsset jio-latest.js, https://lab.nexedi.com/nexedi/jio/raw/master/dist/jio-v3.42.0.js?inline=false
+
+CALL :RecuperationAsset rsvp-2.0.4.min.js, https://lab.nexedi.com/nexedi/rsvp.js/raw/master/dist/rsvp-2.0.4.min.js?inline=false 
+
+exit /B 0
+
+:RecuperationAsset
+IF Exist %~1 (
+    echo "Le fichier %~1 existe déjà"
```
Comment (rigelk) [inline scripts/get-assets-w.bat]: same

File: tsconfig.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
     "declaration": false,
     "module": "esnext",
     "moduleResolution": "node",
-    "emitDecoratorMetadata": true,
+    "emitDecoratorMetadata": false,
```
Comment (rigelk) [inline tsconfig.json]: is this not is a leftover from debug?

File: Dockerfile [hunk 5]
```diff
@@ -1,6 +1,16 @@
+#Build de mute
+FROM node:16-alpine AS builder
+
+WORKDIR /app
+COPY package.json package-lock.json /app/
+RUN npm ci
+COPY . /app
+RUN npm run build
+
+#Lancement de Mute
```
Comment (rigelk) [inline Dockerfile]: comments should be in english

File: Dockerfile [hunk 6]
```diff
@@ -1,6 +1,16 @@
+#Build de mute
```
Comment (rigelk) [inline Dockerfile]: comments should be in english

**Review by rigelk — CHANGES_REQUESTED** [review]

**Comment by BaptisteHubert** [comment]
I have taken the advices in consideration and fixed the files accordingly. Tested with docker-compose and the application runs locally. The signaling server also works (test with different tab).

File: src/environments/environment.prod.ts [hunk 7]
```diff
@@ -1,8 +1,9 @@
 import { IEnvironment } from './IEnvironment.model'
-import { environment as defaultEnvironment } from './environment'
+import { defaultEnvironment } from './default'
+import { LogLevel } from 'netflux'
 import { EncryptionType } from '@app/core/crypto/EncryptionType.model'
 
-const host = 'coedit.re'
+const host = 'localhost'
```
Comment (rigelk) [inline src/environments/environment.prod.ts]:
```suggestion
const host = 'localhost' // FIXME: interpolation at build time required
```

File: src/environments/environment.ts [hunk 8]
```diff
@@ -1,59 +1,8 @@
...
-
-  debug: {
-    visible: true,
-    log: {
-      netflux: [LogLevel.DEBUG],
-      crypto: false,
-      doc: false,
-    },
-  },
+const host = 'coedit.re'
```
Comment (rigelk) [inline src/environments/environment.ts]:
```suggestion
```

**Review by rigelk — CHANGES_REQUESTED** [review]
the `start.sh` script should be placed in the eponym directory. Apart from that, LGTM!

**Pull request merged**